### PR TITLE
Math Magicians App: Add calculation and operation.

### PR DESCRIPTION
**Pull request by smunozmo**
# Documentation

All the documentation is on the README file.

## Updates:

*  Added calculation and operation feature.

## Notes:

✅  - Used  React.
✅  - No linters errors.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,48 +1,63 @@
 import React from 'react';
+// import operate from './operate';
```
Comment (MahmoudBakr23) [inline src/components/Calculator.js]: - [ ] For best practices, please delete this commented out line for cleaner and more optimized code 😊🙏🏻

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @smunozmo 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct 
- Workflow is correct 
- Descriptive PR summary and headline 
- The app is working fine 

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

- [ ] For a completely professional README file, please fix the following 😊🙏🏻
1. The issues page link is broken 
2. Kindly remove the testing section because it's not related to the project
3. In the setup section, please add the CLI `npm run start` and what is used for. 

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

n/a

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @smunozmo, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by smunozmo** [comment]
Thank you @Dipeshtwis @MahmoudBakr23

**Pull request merged**